ankihub/settings.py

```diff
@@ -189,6 +192,7 @@ class PrivateConfig(DataClassJSONMixin):
     schema_to_do_full_upload_for_once: Optional[int] = None
     last_sent_summary_date: Optional[date] = None
     feature_flags: dict = field(default_factory=dict)
+    show_enable_fsrs_remind_again: Optional[bool] = True
```

Comment:
:pick: I think we should rename it to `show_enable_fsrs_reminder`, and move it above the `feature_flags` field. The `feature_flags` field is special in a way, so it seems weird to have it somewhere between other fields.

```suggestion
    show_enable_fsrs_reminder: Optional[bool] = Trueä
    feature_flags: dict = field(default_factory=dict)
```